### PR TITLE
Relicense under CC-BY-SA 4.0.

### PR DESCRIPTION
**Pull request by mjstahlberg**
@illwieckz: I'm not sure how the versioning process works now but I'd like to release this as `Delta 6`, which you seem to have butchered to a boring `2.6.0`. :wink:

### CODE REVIEW

**Comment by illwieckz** [comment]
`2.6` for delta `6` is fine, the `delta` thing is mostly something to please our brains, like sugar. The DPK versioning mechanism does not need extra zero (it means you can do `2.6.1` after `2.6`.

It's not a problem to write `2.6.0` in docs, but the git tag is likely to become `2.6` so the package will be named `map-parpax_2.6.dpk` (then you can do `map-parpax_2.6.1.dpk` for minor changes your think that does not deserve a greater “delta” version.

About alpha, beta and other user custom maps flags and qualifiers, I would like to add a field to the `.arena` file. What would be awesome is to only put a raw levelshot  in `/media` and extend the `.arena` file so while browsing the map list or loading a map, the game prints those informations (including authors) properly. Xonotic map list is a good example for that.

File: about/map-parpax.txt [hunk 1]
```diff
@@ -242,6 +242,10 @@ sounds/parpax/elevator.flac
 ? / Delta 5
     - ?
 
+09.07.2020 / 2.6.0 (Delta 6)
```
Comment (illwieckz) [inline about/map-parpax.txt:245]: Note: maybe it would be better to switch to ISO's YYYY-MM-DD later but there is no need to pollute this PR with that.

**Comment by illwieckz** [comment]
LGTM (obviously :grinning:), this is awesome!!! :partying_face: :tada: :balloon: :clinking_glasses: :100: :birthday: :lollipop: :1st_place_medal: :+1: :woman_cartwheeling: :heart_eyes: :hugs: :fireworks: :muscle: 

Note: it's better to postpone the `2.6` git tag application right on release time to avoid the distributed dpk to be nammed like `map-parpax_2.6+commit-date+commit-ref` in case we find some last-minute fix to do on that map. So, nothing prevents to merge this now.

File: about/map-parpax.txt (hunk 1 above)
Comment (mjstahlberg) [reply about/map-parpax.txt:245]: Sure, just being consistent here.

**Pull request merged**